### PR TITLE
Merge New Features to Dev

### PR DESCRIPTION
**Pull request by nnikolopoulos**
Resolve conflicts and merge new features (coming from **main**) to **dev**.

### CODE REVIEW

File: ifrs17/Report/ParameterReportMutableScopeInteractive.ipynb [hunk 1]
```diff
@@ -154,7 +154,7 @@
                 "\n    async Task<GridOptions> GetPartnerRatingReport()",
                 "\n    {",
                 "\n        await GetStorage().InitializeAsync((Year, Month), ReportingNode, Scenario, CurrencyType);",
-                "\n        var data = await workspace.GetPartnerRatingsReportParametersAsync(GetArgs());",
+                "\n        var data = await workspace.GetCurrentPartnerRatingsReportParametersAsync(GetArgs());",
```
Comment (dcolleoni) [inline ifrs17/Report/ParameterReportMutableScopeInteractive.ipynb:157]: don't we have the equivalent for LockedIn?

File: ifrs17/Report/ReportScopes.ipynb [hunk 2]
```diff
@@ -275,12 +275,12 @@
                 "\n",
                 "\npublic interface LockedBestEstimate: IScope<(ReportIdentity Id, CurrencyType CurrencyType), ReportStorage>, IDataCube<ReportVariable> {",
                 "\n    IDataCube<ReportVariable> LockedBestEstimate => GetScope<FxData>((Identity.Id, Identity.CurrencyType, EstimateTypes.BE)).FxData",
-                "\n        .Filter((\"LiabilityType\", Identity.Id.LiabilityType), (\"EconomicBasis\", EconomicBases.L));",
+                "\n        .Filter((\"LiabilityType\", Identity.Id.LiabilityType), (\"EconomicBasis\", EconomicBases.L), (\"AmountType\", \"!CDRI\"));",
```
Comment (dcolleoni) [inline ifrs17/Report/ReportScopes.ipynb:278]: Dear @tkukuljan, it is possible that we need to carry this change in the client project.

Comment (dcolleoni) [reply ifrs17/Report/ReportScopes.ipynb:278]: In general, treatment of CDR should include CDRI, let's talk about it.

**Review by dcolleoni — APPROVED** [review]

File: ifrs17/Report/ParameterReportMutableScopeInteractive.ipynb (hunk 1 above)
Comment (nnikolopoulos) [reply ifrs17/Report/ParameterReportMutableScopeInteractive.ipynb:157]: A separate PR will be created for the locked in interactive reports.

**Pull request merged**